src/storage/txn/scheduler.rs

```diff
@@ -481,10 +535,62 @@ impl<E: Engine, L: LockManager> Scheduler<E, L> {
             t.saturating_elapsed(),
             "initialized the transaction scheduler"
         );
-        Scheduler {
+
+        let result = Scheduler {
             inner,
             _engine: PhantomData,
-        }
+        };
+        result.start_latch_state_dumping(dump_stop_rx);
+        result
+    }
+
+    pub fn start_latch_state_dumping(&self, mut stop_rx: tokio::sync::mpsc::Receiver<()>) {
+        let self1 = self.clone();
+        self.inner
+            .high_priority_pool
+            .pool
+            .spawn(async move {
+                loop {
+                    tokio::select! {
+                        msg = stop_rx.recv() => {
+                            assert!(msg.is_none());
+                            break;
+                        }
+                        _ = tokio::time::sleep(Duration::from_secs(60)) => (),
+                    }
+
+                    let _g = self1.inner.latch_state_dump_lock.write();
+
+                    let formatted_latch_state = self1.inner.latches.dump_string();
+
+                    let mut formatted_tasks = String::new();
+                    for (slot_index, slot) in self1.inner.task_slots.iter().enumerate() {
+                        let slot = slot.lock();
+                        if slot.is_empty() {
+                            continue;
+                        }
+
+                        writeln!(formatted_tasks, "Slot {}:", slot_index).unwrap();
+                        for (key, item) in slot.iter() {
+                            writeln!(
+                                formatted_tasks,
```

Comment:
We may need a way to know the current task executing status, especially for those holding latches for a long time, like:
- If the task is being scheduled or it's already being decided to be canceled because of timeout.
- If the task is being executed, what is it doing now, is it waiting for the result of `async_write`, how much time does it take to finish `process_write` for a command, and when it sends the writing task to `raftstore`, is its proposal already proposed or committed? As we've just encountered the suspicious 3-hour `async_write` processing.

@MyonKeminta 
It may not be easy to implement, I think we could think about it first?

Reply:
To be able to be used in production, I'm considering about collecting more information (such as the time cost of each phases in scheduler) with the tracker and put them to the `WriteDetail`. Have we ever consider about this?

Reply:
https://github.com/tikv/tikv/pull/14476
Sry, we already have it in master and 7.1, but it's not in release-6.5 branch. Does it help if we pick it to release-6.5